src/views/v2/month/mobile-event.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * View: Month View Mobile Event
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/views/v2/month/mobile-event.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @version TBD
+ *
+ */
+
+$event    = $this->get( 'event' );
+$event_id = $event->ID;
+
+$classes = [ 'tribe-events-calendar-month__mobile-event' ];
+
+/* @todo fix this once we make event dynamic */
+// if ( tribe( 'tec.featured_events' )->is_featured( $event_id ) ) {
+	$classes[] = 'tribe-events-calendar-month__mobile-event--featured';
+// }
+?>
+
+<article class="<?php echo esc_attr( implode( ' ', $classes ) ); ?>">
```

Comment:
Do we need aria grid atts for the mobile implementation or not necessary?

Reply:
Nope, this is one of the events that shows up below the calendar when a day is pressed.